internal/pkg/gateway/commit/notifier.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package commit
+
+import (
+	commonProto "github.com/hyperledger/fabric-protos-go/common"
+	ordererProto "github.com/hyperledger/fabric-protos-go/orderer"
+	peerProto "github.com/hyperledger/fabric-protos-go/peer"
```

Comment:
I just want to come back around an highlight that these are not typical import names for Go. We avoid camel case for the package names.

Can se please revisit these imports and use consistent, lower case import names when needed?

Reply:
+1 not sure how I missed this